djcelery/tests/test_commands.py

```diff
@@ -14,11 +14,11 @@ def test_celeryd_command():
     else:
         traceback = None
     with patch(CELERYD_COMMAND) as handle:
-        call_command('celeryd')
+        call_command('celeryd', '--hostname=test', '--loglevel=WARN')
```

Comment:
`execute_from_command_line(['manage.py', 'celeryd', '--hostname=test', '--loglevel=WARN'])`
And remove `skip_checks=True` below.